self-core-impl/src/main/java/com/selfxdsd/core/GithubRepo.java

```diff
@@ -122,7 +122,11 @@ public Webhooks webhooks() {
     public Stars stars() {
         return new GithubStars(
             this.resources(),
-            this.repoUri(),
+            URI.create(this
+                .repoUri()
+                .toString()
+                .replace("repos", "user/starred")
+            ),
```

Comment:
@amihaiemil Here was a bug - was passed the repo url of  `/user/starred/:owner/:repo`